src/movies.js

```diff
@@ -1,16 +1,58 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors() {}
-
+function getAllDirectors(arr) {
+  const pickDirector = arr.map((x) => x.director);
+  return pickDirector;
```

Comment:
If the code is this short, you can alway return result directly - no need to create variable

```
return arr.map((x) => x.director)
```